components/gc-toc/toc-en.html

```diff
@@ -0,0 +1,57 @@
+---
+{
+	"title": "GCWeb Table of Contents example",
```

Comment:
Don't need to specify "GCWeb" here

```suggestion
	"title": "In-page table of contents",
```